docs/modules/ROOT/pages/includes/equals-hashcode.adoc

```diff
@@ -0,0 +1,9 @@
+`Hashcode` and `equals` are by default generated. It adds all the variables in the model
+to the model to the methods.
```

Comment:
```suggestion
`Hashcode` and `equals` are by default generated. It adds all the variables in the model to the model to the methods.
```
I am not sure if I understood this sentence. 🤔

Reply:
In my head it was 100% clear when I pushed it :D I will try and refrase it! :)

Reply:
I rewrote the documentation, I hope this ones better.